peps/pep-0007.rst

```diff
@@ -135,6 +135,22 @@ Code lay-out
           return 0; /* "Forgive" adding a __dict__ only */
       }
 
+* Vertically align line continuation characters in multi-line macros.
+
+* Macros intended to be used as a statement should use the
+  ``do { ... } while (0)`` macro idiom,
+  without a final semicolon.
+  Example::
+
+      #define ADD_INT_MACRO(MOD, INT)                                   \
+          do {                                                          \
+              if (PyModule_AddIntConstant((MOD), (#INT), (INT)) < 0) {  \
+                  goto error;                                           \
+              }                                                         \
+          } while (0)
+
+* ``#undef`` private macros after use.
```

Comment:
Perhaps elaborate on what you mean by "private macro"?

Reply:
How about this?

```suggestion
* ``#undef`` file local macros after use.
```